### DISCUSSION TITLE
If you were using a tandoor in Indian cooking, what would it be?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#85] [Galaxy Trivia] [answered]
**Hint:** 🔥

**Comment by Tynab** [comment, marked as answer]
If you were using a tandoor in Indian cooking, it would be a traditional cylindrical clay oven used for cooking and baking in Indian and Pakistani cuisine. Tandoors are typically used for cooking dishes such as naan bread, tandoori chicken, and various kebabs. The food is cooked in the tandoor at high temperatures, typically using charcoal or wood as the heat source, and the clay walls of the tandoor help to distribute the heat evenly for a unique and delicious flavor.

**Comment by oscarxdev** [comment]
A tandoor is a traditional cylindrical clay oven used in Indian cooking. It is heated with charcoal or wood to high temperatures, and is used for cooking a variety of foods, including breads, meats, and vegetables. The heat from the charcoal or wood creates an intense radiant heat that cooks the food quickly and evenly, while also giving it a distinctive smoky flavor. Tandoors are a common feature in Indian and Pakistani restaurants, and are also used in home cooking in some parts of India. Some of the popular dishes cooked in a tandoor include tandoori chicken, naan bread, and kebabs.
